docs_app/content/tutorial/6-chaining-operators.md

```diff
@@ -0,0 +1,145 @@
+# 6. Chaining Operators
+
+## TLDR:
+
+- `pipe` is a functional programming concept that executes functions in order, passing the result of each one to the next.
+- Use the `Observable` [`pipe`](API) method to chain operators together and build new Observables.
+- You can create many small chains to shorten larger chains.
+
+## Overview
+
+One of central points of RxJS are combining its many operators in interesting and useful ways. This is done by [chaining](GL) the operators together using functional programming tricks built into RxJS for convenience.
+
+If an operator, as stated in ["what is an operator"](4-what-is-an-operator.md#Operators), is a "thing that can perform an operation", on a collection, then we might be able to go back to our [collection of apples example](4-what-is-an-operator.md#Collections) from that section.
```

Comment:
Consider removing the first comma for clarity: "operation", on a collection," -> "operation" on a collection,"